module-4/backend/week_two.md

```diff
@@ -4,8 +4,59 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+1. What's one difference between ES5 and ES6?
+
+ES6 is moving JS to more natively support OOP.  One instance of this is the new Class structure.
+
+2. What's the difference between asynchronous and synchronous JavaScript? 
+
+synchonous JS runs all at once where as asynchronous allows us to tell the program to run Y line of code after X has happened successfully.
```

Comment:
I'd be careful with the `all at once` description. Sync is still sequential, just on a single thread.